packages/aws-cdk/README.md

```diff
@@ -382,6 +382,30 @@ $ cdk deploy --method=prepare-change-set --change-set-name MyChangeSetName
 For more control over when stack changes are deployed, the CDK can generate a
 CloudFormation change set but not execute it.
 
+#### Import existing resources
+
+You can pass the `--import-existing-resources` flag to the `deploy` command:
+
+```console
+$ cdk deploy --import-existing-resources
+```
+
+This allows you to automatically import the resources in your template that already exist in your AWS account during CloudFormation deployments. With this feature, you can reduce the manual effort of import operations and avoid deployment failures because of naming conflicts.
+
+To review which resources are imported or not before actually executing a change set, use `--method=prepare-change-set` flag. You can inspect the change set created by CDK from the management console or other external tools.
```

Comment:
```suggestion
This allows you to automatically import the resources in your template that already
exist in your AWS account during CloudFormation deployments.
With this feature, you can reduce the manual effort of import operations and avoid
deployment failures because of naming conflicts.

To review which resources are imported or not before actually executing a change set,
use `--method=prepare-change-set` flag.
You can inspect the change set created by CDK from the management console or other external tools.
```
Formatting